.github/workflows/main.yml

```diff
@@ -12,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```

Comment:
This should probably be the same as defined in package.json.
https://github.com/JsDaddy/ngx-mask/blob/b9968ae7cf27616678c487c67daa8de8088fe9a3/package.json#L6-L8